_posts/2021-03-29-Arts and Sciences Recipe Page.md

```diff
@@ -0,0 +1,145 @@
+
+* writing tags:
+  * generic tags:
+    * \<abr type = *full word*> *abbreviation* \</abr> used to indicate an abbreviation for a longer full word.
+    * \<pl> *place* \</pl> used to indicate a word representing a place name
+    * \<person> *person name* \</person> used to indicate a word/phrase representing a person's name and/or title
+  * recipe specific tags:
+    * \<qt> *quantity* \</qt> to indicate a word/phrase representing a quantity of material in the recipe
+    * \<m> *material* \</m> to indicate a word/phrase representing a material used in the recipe
+    * \<tl> *tool* \</tl> to indicate a tool used in the recipe
+  * document specific tags:
+    * \<recNum> *recipeNum* \<recNum> to indicate that a number was the number of the recipe in the list (these numbers were marked on the document by the author)
+    * \<pageNum> *pageNum* \<pageNum> to indicate a physically marked page number. Note that these numbers don't line up with the page in the online manuscript due to blank pages and front matter at the beginning. For that reason and just to let researchers know that the author did in fact put page numbers, I felt that putting this tag in addition to the page tag was worth it.
+
+### What textual structure seems appropriate? Consider the hierarchy of your tags.
+
+On a broad organizational level, the document was separated as a recipe list, with a number, title, and (sometimes) multiple paragraphs of writing in each recipe. In addition, there were page numbers and breaks that frequently appeared in the middle of recipes (but not paragraphs, since that doesn't really make sense).
+
+To account for this organization, I broadly formatted my text with paragraph tags, page tags, recipe number tags, and heading tags nested directly inside div tags (with type = "recipe"). Margin text occasionally appeared alongside title content outside of paragraphs, so there are margin tags nested directly inside div tags as well. Also, some lines or double lines appear between titles and paragraphs, so these tags are also nested directly inside div tags. (double lines also appear outside of recipes, so these are the only tag put outside of div tags).
+
+In terms of finer textual structure, there was some tension about  hierarchy when physical aspects of the texts. For example, there were chunks of text including headings/titles, recipe numbers, lines, and margin writing that were all a different ink color (red or brown instead of black). In these cases, I chose to put color tags as the children of other writing or physical tags (aside from the line or double line tags since these are marks themselves rather than indicators of a physical property of enclosed writing).
```

Comment:
This tension within the hierarchy of your tags is _always_ something to work through in XML. But I think what you've worked out here is a very good structure that takes into account the physical properties of the recipe document, while also making legible to a researcher that different colored inks represent different instances of writing.